doc/design.md

```diff
@@ -214,23 +214,24 @@ Needs: impl, itest
 
 ### Shortened Docker Image References
 
-`dsn~shortened-docker-image-references~1`
+`dsn~shortened-docker-image-references~2`
 
 The `ExasolDockerImageReference` can be constructed with the following forms of shortened image references as parameter:
 
-* `<major>`
-* `<major>.<minor>`
-* `<major>.<minor>.<fix>`
-* `<major>.<minor>.<fix>-d<docker-image-revision>`
+* `<major>[-suffix][-d<docker-image-revision>]`
+* `<major>.<minor>[-suffix][-d<docker-image-revision>]`
+* `<major>.<minor>.<fix>[-suffix][-d<docker-image-revision>]`
```

Comment:
What happens if the -suffix starts with -d ?

Reply:
Made that clearer in the requirement that the distinguishing factor is that `-d` followed by numbers constitutes the image revision. Also it needs to come last as seen in the spec.

To make sure this works, I added more test cases.